README.md

```diff
@@ -7,16 +7,16 @@ This repo provides an open source app called Offline Mapbook built in WPF with t
 - [Features](#features)   
 - [Detailed Documentation](#detailed-documentation)   
 - [Development Instructions](#development-instructions)   
-   - [Fork the repo](#fork-the-repo)   
-   - [Clone the repo](#clone-the-repo)   
-      - [Command line Git](#command-line-git)   
-      - [Configuring a Remote for a Fork](#configuring-a-remote-for-a-fork)   
```

Comment:
I would suggest that these go back to being h3s/H4s (they feel like sub-steps of 'Development Instructions', so no change necessary), but I agree with what you've done with 'Requirements' and below.